docs/source/index.md

```diff
@@ -0,0 +1,45 @@
+# Caliper
+
+Caliper is a blockchain performance benchmark framework, which allows users to test different blockchain solutions with predefined use cases, and get a set of performance test results.
+
+## Supported Blockchain Solutions
+
+Currently supported blockchain solutions:
+
+- [Hyperledger Besu](https://github.com/hyperledger/besu)
+- [Ethereum](https://github.com/ethereum/)
+- [Hyperledger Fabric](https://github.com/hyperledger/fabric)
+- [FISCO BCOS](https://github.com/FISCO-BCOS/FISCO-BCOS)

```

Comment:
Still shows FISCO BCOS. Was this built from the v0.6.0 docs ?

Reply:
I forgot to remove it.
I have made changes in the 0.5.0 docs code so it remained. Extremely sorry